EssentialCSharp.Web/wwwroot/css/styles.css

```
@@ -52,16 +250,16 @@ body {
 }
 
 @media only screen and (max-width: 768px) {
-  .page-nav-container {
+  .turn-page {
     top: calc(100vh - 13rem);
```

```suggestion
    top: calc(100vh - 13rem);
    top: calc(100dvh - 9rem);
```
I realized we should use dvh here for browsers that support it. And if the browser doesnt then the top with the vh will be used.

---

Order is important here. Top with dvh needs to override top with vh.